### ISSUE TITLE
Handle bigint data types correctly

### ISSUE THREAD

**Issue by rohilsurana** [BemiHQ/BemiDB#5]
Currently `bigint` Postgres data type is getting mapped to 32 bit integer data types, `int` and `int32` for Parquet and Iceberg respectively. Instead it should be mapped to 64 bit integer given that the underlying data type for `bigint` is 8 byte `int8`.

**Comment by arjunlol** [comment]
Thanks @rohilsurana, we're taking a look and should have a fix shortly!

**Comment by exAspArk** [comment]
Thank you, @rohilsurana!

This should now be resolved in v0.3.2. Would you be able to [install](https://github.com/BemiHQ/BemiDB#quickstart) the latest version and try it out?

---

I'll paste some test data and an error stacktrace for visibility:

```sql
CREATE TABLE bigint_example (
    id SERIAL PRIMARY KEY,
    big_number BIGINT
);

INSERT INTO bigint_example (big_number) VALUES 
    (9223372036854775807),  -- Max value for BIGINT
    (-9223372036854775808); -- Min value for BIGINT
```

```
panic: Write error: integer overflow on token 9223372036854775807

goroutine 1 [running]:
main.PanicIfError({0x102e43ec8?, 0x14000416660?}, {0x0?, 0x140001e1400?, 0x2?})
	/Users/exaspark/Documents/Code/BemiHQ/BemiDB/src/utils.go:14 +0x9c
main.(*IcebergWriter).Write(0x1400000e5a0, {{0x14000480010?, 0x14000480010?}, {0x14000408fe0?, 0x14000408fe0?}}, {0x140001e1400, 0x2, 0x2}, 0x1400000e7b0)
	/Users/exaspark/Documents/Code/BemiHQ/BemiDB/src/iceberg_writer.go:339 +0x134
main.(*Syncer).syncFromPgTable(0x1400000e5d0, 0x14000422b40, {{0x14000480010?, 0x6?}, {0x14000408fe0?, 0x0?}})
	/Users/exaspark/Documents/Code/BemiHQ/BemiDB/src/syncer.go:108 +0x304
main.(*Syncer).SyncFromPostgres(0x1400000e5d0)
	/Users/exaspark/Documents/Code/BemiHQ/BemiDB/src/syncer.go:46 +0x1a8
main.syncFromPg(0x1034e38a0)
	/Users/exaspark/Documents/Code/BemiHQ/BemiDB/src/main.go:60 +0x24
main.main()
	/Users/exaspark/Documents/Code/BemiHQ/BemiDB/src/main.go:25 +0xb0
exit status 2
Error: error running script "cd src && go run . sync" in Devbox: exit status 1
```
